test/client.test.js

```diff
@@ -123,6 +123,38 @@ describe('Client', () => {
     await client.ping()
     expect(fetch).toBeCalled()
   })
+
+  test('set queryTimeout on client instance', () => {
+    client.queryTimeout(1000)
+    expect(client._queryTimeout).toEqual(1000)
+
+    client.queryTimeout(2000)
+    expect(client._queryTimeout).toEqual(2000)
+  })
```

Comment:
Test private fields are a bad test pattern since it is testing the implementation and not the interface.